src/Commands/GenerateTypesCommand.php

```diff
@@ -92,6 +97,7 @@ public function __construct()
     public function handle(): void
     {
         $this->modelDir = $this->option('modelDir') ?? config('laravel-model-ts-type.model_dir');
+        $this->namespace = $this->option('namespace') ?? config('laravel-model-ts-type.namespace');
```

Comment:
Could you also add this command option to the README, and a default `false` to the config file?

Reply:
Same here 😊